frontend/src/components/table/schedule/use-schedule-state.ts

```diff
@@ -22,11 +26,11 @@ export function useScheduleState(): useScheduleStateReturn {
     (scheduleModel: ScheduleDataModel): void => {
       const logic = new ScheduleLogic(scheduleModel, dispatchGlobalState);
       setScheduleLocalState({
-        nurseShiftsSection: logic.getNurseInfo().sectionData,
-        babysitterShiftsSection: logic.getBabySitterInfo().sectionData,
-        childrenSection: logic.getChildrenInfo().sectionData,
-        extraWorkersSection: logic.getExtraWorkersInfo().sectionData,
-        dateSection: logic.getMetadata().sectionData,
+        nurseShiftsSection: (logic.sections.NurseInfo as ShiftsInfoLogic).sectionData,
```

Comment:
Wiem, że  trochę brzydko to wygląda, ale nie chciałem sie nad tym zastanawiać, bo jeszcze nie wiem, czy nie usuniemy tego stanu.